### PR TITLE
Update ScriptHandler for composer installs

### LINKED ISSUE #2450: Composer install : command line interface
I'm starting a new issue as a previous issue #2309 I opened was targeting multiple problems. 

The post install script does nothing about the app dir. Shouldn't the config, database, cache, resources directories be created and filled with files accordingly ?
When installing the web dir outside the root, there's no command line utility (nut) available.

### PR DESCRIPTION
**Pull request by nlemoine**
Fixes #2450

### CODE REVIEW

File: src/Composer/ScriptHandler.php [hunk 1]
```diff
@@ -43,7 +40,7 @@ public static function installAssets(CommandEvent $event, $options = false)
         $filesystem->remove($targetDir);
         $filesystem->mkdir($targetDir, $dirMode);
 
-        foreach (['css', 'fonts', 'img', 'js'] as $dir) {
+        foreach (array('css', 'fonts', 'img', 'js') as $dir) {
```
Comment (GawainLynch) [inline src/Composer/ScriptHandler.php]: PHP 5.4+ array syntax please

File: src/Composer/ScriptHandler.php [hunk 2]
```diff
@@ -56,61 +53,95 @@ public static function installAssets(CommandEvent $event, $options = false)
...
      */
-    protected static function getOptions(CommandEvent $event)
+    protected static function getOptions(Event $event)
     {
         $options = array_merge(
-            [
+            array(
                 'bolt-web-dir'  => 'web',
                 'bolt-app-dir'  => 'app',
                 'bolt-dir-mode' => 0777
```
Comment (GawainLynch) [inline src/Composer/ScriptHandler.php]: PHP 5.4+ array syntax please

**Comment by GawainLynch** [comment]
You will need to rebase that merge out too please.

File: src/Composer/ScriptHandler.php (hunk 1 above)
Comment (nlemoine) [inline src/Composer/ScriptHandler.php]: That was in my scope but I'll do that ;)

Comment (GawainLynch) [inline src/Composer/ScriptHandler.php]: No stress… apologies if that came out rudely, I worked though the night am was in automatic :grin:

**Comment by nlemoine** [comment]
> You will need to rebase that merge out too please.

PR from master?

**Comment by GawainLynch** [comment]
Oh… it was master… you don't need to do anything then. Did I mention my current lack of sleep? :wink:

**Comment by nlemoine** [comment]
I made the PR into bolt master from my fix-composer-install branch. 

Not very familiar with rebasing.

**Comment by GawainLynch** [comment]
No problems… it is a great one to learn, but if you're not familiar with it, you can break things on yourself.

**Comment by bobdenotter** [comment]
This is good to go, right?

**Comment by GawainLynch** [comment]
@bobdenotter no. @rossriley has concerns as the two PRs some some problems but close of other things and we need to figure out what is the best approach.

**Comment by bobdenotter** [comment]
Pinging @rossriley @nlemoine 

What do we need to do to get this PR mergable? :-)

**Comment by nlemoine** [comment]
Most issues raised by composer install can be found in ticket #3551. Please check it out. On holidays right now, i'll have a look at this when i'm back!

**Comment by GawainLynch** [comment]
Enjoy your holiday, @nlemoine :+1:

**Comment by nlemoine** [comment]
Hi @GawainLynch, @rossriley, I'm back from holidays. Did you have a look at https://github.com/bolt/bolt/issues/3551? Most issues concerning composer install of Bolt are gathered here.

**Comment by GawainLynch** [comment]
Welcome back @nlemoine… I hope you had a good holiday.

It has been a while since I had a chance to talk to @rossriley about this as we've been extremely busy with the storage work, but he did raise some concerns as to approach… I'll let him chime in though as I can't remember with specificity.

**Pull request closed without merging**